src/tm-threads.c

```diff
@@ -248,7 +283,7 @@ static void *TmThreadsSlotPktAcqLoop(void *td)
                                  " tmqh_out=%p",
                    s, s ? s->PktAcqLoop : NULL, tv->tmqh_in, tv->tmqh_out);
         TmThreadsSetFlag(tv, THV_CLOSED | THV_RUNNING_DONE);
-        pthread_exit((void *) -1);
+        TmThreadExit(tv, -1);
```

Comment:
Initial change should go into its own commit, then add the specific dpdk changes

Reply:
done